charts/rancher-vsphere-csi/values.yaml

```diff
@@ -156,7 +161,7 @@ versionOverrides:
       csiController:
         image:
           repository: rancher/mirrored-cloud-provider-vsphere-csi-release-driver
-          tag: v2.5.3
+          tag: v2.5.2
```

Comment:
Why are you bumping the versions down from 2.5.3->2 2.4.3->2, shouldn't the version numbers go up? 2.5.4 for the driver and syncer are released. I'm already going to update this in https://github.com/rancher/vsphere-charts/pull/35, I'm just wondering why we're backtracking the versions here

Reply:
most likely this was a mistake or a value I got when I forked the repo to add labels.  Please fix this to what it needs to be.

Reply:
updated to match all version updates

Reply:
files changes for values now does not have any version differences in them